media/css/legal/builders-challenge.scss

```diff
@@ -0,0 +1,40 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/protocol/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+.builders-challenge-terms {
+    ol > li::marker {
+        font-weight: bold;
+    }
+
+    .nested-lists {
+        counter-reset: count;
+
+        .nested-counter-list {
+            li {
+                counter-increment: count 1;
+
+                &::marker {
```

Comment:
**issue (potentially blocking)**: IE doesn't support `::marker` styles

**suggestion**: Check if Legal is OK with the default alpha-ordered markers. These will show on IE browsers regardless.
<img width="68" alt="ie-alpha" src="https://user-images.githubusercontent.com/19650432/228184844-685df8b5-47b0-40f6-a0d8-5e1f6e7dce31.png">